src/generate_trigger.ts

```diff
@@ -0,0 +1,112 @@
+import { parse } from "./deps.ts";
+
+export type WorkflowRecord = {
+  "callback_id": string;
+};
+
+interface Inputs {
+  [key: string]: { value: string };
+}
+
+export type TriggerFileRecord = {
+  "type": string;
+  "name": string;
+  "description"?: string;
+  "workflow": WorkflowRecord;
+  "inputs"?: Inputs;
+};
+
+export type TriggersPayload = {
+  triggers: TriggerFileRecord[];
+};
+
+interface Response {
+  ok: boolean;
+  files: Array<string>;
+  error?: {
+    message: string;
+  } | null;
+}
+
+export const generateTriggerFile = async (
+  input: string[],
+): Promise<Response> => {
+  // Parse JSON payload
+  const source = parse(input).source as string;
+  if (!source) throw new Error("A source object needs to be defined");
+  const payload: TriggersPayload = JSON.parse(source);
+
+  // Create the triggers directory if it does not exist
+  const directory = "triggers";
+  try {
+    await Deno.stat(directory);
+  } catch (error) {
+    if (error instanceof Deno.errors.NotFound) { // create directory
+      await Deno.mkdir(directory);
+    } else {
+      throw error;
+    }
+  }
+
+  // Generate the files
+  const files: Array<string> = [];
+  await Promise.all(payload.triggers.map(async (tr: TriggerFileRecord) => {
+    const templateString = SlackTriggerTemplate(tr);
+    const filename = directory + `/${tr.workflow.callback_id}.ts`;
+
+    await Deno.writeTextFile(filename, templateString);
+    files.push(filename);
+  }));
+
+  return { ok: true, files: files };
+};
+
+const autogeneratedComment = () => {
+  const time = new Date();
+  return `/** This file was autogenerated on ${time.toDateString()}. **/`;
+};
+
+export const SlackTriggerTemplate = (tr: TriggerFileRecord) => {
+  const camelCase = tr.workflow.callback_id.toLowerCase().replace(
+    /[^a-zA-Z0-9]+(.)/g,
+    (_, chr) => chr.toUpperCase(),
+  );
+
+  let inputValues = ``;
+  // Add interactivity param to trigger inputs if present
+  if (tr.inputs) {
+    for (const prop in tr.inputs) {
+      if (tr.inputs[prop].value.includes("interactivity")) {
+        inputValues += `
+  ${prop}: {
+    value: "{{data.interactivity}}",
+  },
+`;
```

Comment:
What determines if the trigger is going to have an `inputs` object and if that value has `interactivity` in it? It'd be nice to have the whole workflow available so we can determine whether or not an input is of type interactivity but maybe the CLI is doing that.